test/acceptance/acceptance_test.go

```diff
@@ -0,0 +1,57 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build !windows
+
+package acceptance
+
+import (
+	"github.com/aws/amazon-cloudwatch-agent-test/environment"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/acceptance/testrunners"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/status"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/test_runner"
+	"github.com/stretchr/testify/suite"
+	"testing"
+)
+
+var envMetaDataStrings = &(environment.MetaDataStrings{})
+
+func init() {
+	environment.RegisterEnvironmentMetaDataFlags(envMetaDataStrings)
+}
+
+type AcceptanceTestSuite struct {
+	suite.Suite
+	test_runner.TestSuite
+}
+
+var _ test_runner.ITestSuite = (*AcceptanceTestSuite)(nil)
+
+func (suite *AcceptanceTestSuite) GetSuiteName() string {
+	return "SecurityTestSuite"
+}
+
+var (
+	ec2TestRunners []*test_runner.TestRunner
+)
+
+func getEc2TestRunners() []*test_runner.TestRunner {
+	if ec2TestRunners == nil {
+		ec2TestRunners = []*test_runner.TestRunner{
+			{TestRunner: &testrunners.FilePermissionTestRunner{}},
+		}
+	}
+	return ec2TestRunners
+}
+
+func (suite *AcceptanceTestSuite) TestAllInSuite() {
+	for _, testRunner := range getEc2TestRunners() {
+		testRunner.Run(suite)
+	}
+
+	suite.Assert().Equal(status.SUCCESSFUL, suite.Result.GetStatus(), "Security Test Suite Failed")
+}
+
+func TestAcceptanceTestSuite(t *testing.T) {
+	suite.Run(t, new(AcceptanceTestSuite))
+}
```

Comment:
It would be a less confusing to follow the Golang defined mechanisms for testing.
As in just put each test case in a function named `TestXXX`, e.g. `TestFilePermission`, or `Test

1 big problem with the current "test suite" is that developers cannot easily run a single test case on their local ec2 instance. I very much want to just SSH into my EC2 instance and run:
`go test packageName -run NameOfTest`


Today, if I want to run the "cpu" related integration tests I must comment out code from the test suite.
It isn't a huge problem, but I would like to avoid the same problem again.


Can you think of any ways to simplify the implementation down to a set of test cases that can be individually run without modifying test code? e.g. `go test packageName -run NameOfTest`. 

If we make our test look and behave like other go tests it will be easier for future devs to understand.

Reply:
thinking about how to address this